src/components/ShowMenu.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+export class ShowMenu extends Component {
+  state = {
+    menuList: [],
+  };
+
+  componentDidMount() {
+    let headers = {
```

Comment:
do not think you need to specify these headers if you do not add anything else to them